treeherder/perf/fixtures/performance_bug_templates.yaml

```diff
@@ -2,10 +2,10 @@
   pk: 1
   fields:
     framework: 1
-    keywords: perf, regression, talos-regression
-    default_component: Performance
+    keywords: perf, perf-alert, regression, talos-regression
+    default_component: General
     default_product: Testing
-    cc_list: dave.hunt@gmail.com, rwood@mozilla.com, igoldan@mozilla.com, fstrugariu@mozilla.com, alexandru.ionescu@softvision.ro, marian.raiciof@softvision.ro
+    cc_list: dave.hunt@gmail.com, rwood@mozilla.com, igoldan@mozilla.com, fstrugariu@mozilla.com
```

Comment:
I suspect you're on an older revision, as these were recently added.

Reply:
Oh, I forgot to rebase the currrent git branch. My bad.